x-pack/plugins/fleet/common/types/models/epm.ts

```diff
@@ -50,6 +50,7 @@ export enum KibanaAssetType {
   indexPattern = 'index_pattern',
   map = 'map',
   lens = 'lens',
+  mlModule = 'ml_module',
```

Comment:
I reckon it'd better to store all these string literals like `ml_module `, `ml-module`, `ML Module`, and `mlApp` in some constant and expose from the ML plugin if possible.